mods/tuxemon/db/technique/status_dozing.json

```diff
@@ -18,7 +18,7 @@
     "item": 0
   },
   "tech_id": 0,
-  "use_failure": "generic_failure",
+  "use_failure": "combat_state_dozing_get",
```

Comment:
it's because in that context status_dozing is a technique and not a status;

that's because status_dozing creates a null action (no action), so it passes the status_dozing itself, but it fails (because 0 power, etc), then we need the **combat_state_dozing_get** (equal to XXX is asleep).

when the status checks, the it looks at success (the standard text that appears every turn (like poison).

The best way to handle this would be allow None (Technique) in CombatState, but it would be another issue at the moment.